README.md

```diff
@@ -92,10 +94,20 @@ patron_response = api_client.fetch_user(None, {
 which you can use to extract [pagination links](http://jsonapi.org/format/#fetching-pagination)
 from our [json:api response documents](http://jsonapi.org):
 ```python
-api_client = patreon.API(patron_access_token)
-patrons_page = api_client.fetch_page_of_pledges(campaign_id, 10)
-next_cursor = api_client.extract_cursor(patrons_page)
-second_patrons_page = api_client.fetch_page_of_pledges(campaign_id, 10, cursor=next_cursor)
+api_client = patreon.API(creator_access_token)
```

Comment:
looks like we might want to update https://docs.patreon.com/?python#paging-through-a-list-of-pledges-to-you

Reply:
yupyup! once all this stuff is done, we'll be updating all docs on docs.patreon.com :)